index.ts

```diff
@@ -0,0 +1,308 @@
+import express = require("express");
+import http = require("http");
+import handlebars = require("express-handlebars");
+import popsicle = require('popsicle');
+import ClientOAuth2 = require('client-oauth2');
+import path = require("path");
+
+import morgan = require("morgan");
+import websocket = require("websocket");
+import bodyParser = require("body-parser");
+import dotenv = require("dotenv");
+import session = require("express-session");
+import {Express} from "express";
```

Comment:
`express` is imported twice here, see first line

Reply:
The second import was necessary for the type annotation. Since I still haven't completely understood the import system, I am happy for clarifications.